docs/javascript/v2/debugging/debugging.mdx

```diff
@@ -56,13 +55,15 @@ on [discord](https://100ms.live/discord), there is a channel there for web-sdk-d
 If you're reporting a bug, we would appreciate as many details possible. We might further ask you for the following information to debug the issues better -
 
 ### Steps to reproduce
-Detailed steps to reproduce the issue. You can also give us a [codesandbox](https://codesandbox.io/) where the issue can be seen, feel free
+
+Detailed steps to reproduce the issue. You can also give us a <a onClick={()=>{window.analytics.track("codesandbox.viewed",{path:window.location.pathname});window.open("https://codesandbox.io","_blank")}}>codesandbox</a> where the issue can be seen, feel free
```

Comment:
why not have an override for anchor elements similar to code snippets ? 
why is the logic of firing events exposed here  ? 

we should make adding events simple for people contributing to this repo

Reply:
We cant pass function as well as long string text or string seperated by "."

Reply:
we dont need to pass functions right 🤔 ?